### DISCUSSION TITLE
UI, Documentation and general feedback

### DISCUSSION THREAD

**Discussion by Revenger** [OpenI6X/opentx#363] [Ideas]
I got the firmware installed and revereted for now due to the channel 5/6 USB bug.

I have some feedback as a new user with getting a transmitter and trying this firmware.

You may not be able to change this due to it possinbly being a fork of OpenTX but the menu UI is a bit complex and should probably be tweaked to be better and similar to the default ui control system where possible.
Also there is no documentation here with screenshots for the UI for this transmitter, I was getting pretty lost in the menus and didn't know what a lot of the functionms are.

The thre builds
- ExpressLRS config
- Helicopter
- Default

There should be a table showing the diffrences betwen versions with a check shgowing what comes with what build similar to what you see with software normal, advanced, pro etc.
I had no idea what to chose as there is no documentation about what i get with each.

The Wiki has the flashing guide but main has page has the readme and other documentation, then I had to go to youtube videos to find basic info not even documented on this github.
All documentation should be consolidated in the Wiki with links to each thing and back tabs.
- Features
- Flashing
- Avaliable Hardware Modifications
- USB
- ETC

This saves people swapping back and forth trying to decide if the info they needed was on the wiki or the main readme.

Also if the OpenTX has a companion app couldn't we have a PC settings app also for easier configuration etc?

Also a list of central list of current bugs like the 5/6 issue with the system would be good.

Thanks for reading ansd hope this feedback helps the project going forward.

**Comment by ajjjjjjjj** [comment]
> You may not be able to change this due to it possinbly being a fork of OpenTX but the menu UI is a bit complex and should probably be tweaked to be better and similar to the default ui control system where possible.
Also there is no documentation here with screenshots for the UI for this transmitter, I was getting pretty lost in the menus and didn't know what a lot of the functionms are.

This is the OpenTX port, not a custom firmware. See the OpenTX/EdgeTX manuals all over the internet. There is no radio specific manual due to the menus being almost the same for every radio.

 > There should be a table showing the differences betwen versions with a check shgowing what comes with what build similar to what you see with software normal, advanced, pro etc.
I had no idea what to chose as there is no documentation about what i get with each.

There is simple explaination for every version mentioned. What exactly are you missing?
Upcoming release will have a table with features details due to more builds available.

> The Wiki has the flashing guide but main has page has the readme and other documentation, then I had to go to youtube videos to find basic info not even documented on this github.

Main page readme.md links to wiki's flashing and development guide. If you are missing something - name it.

> All documentation should be consolidated in the Wiki with links to each thing and back tabs.

I don't agree. I prefer to keep documentation versioned in repository.

> Also if the OpenTX has a companion app couldn't we have a PC settings app also for easier configuration etc?

If someone adds it, yes. This is opensource repository, I work on it in my free time, everyone is invited to contribute.

> Also a list of central list of current bugs like the 5/6 issue with the system would be good.

There is https://github.com/OpenI6X/opentx/issues tab in the repository that you can read before flashing to know the USB joystick problems.

**Comment by Revenger** [comment]
Could you please link me to where I can see the actual diffrences in features between these builds and what each one does diffrent from each other and why I should choose one over the other.

openi6x_1.9.0_elrsv3.zip for ExpressLRS config tool.
openi6x_1.9.0_heli.zip if you fly helicopters.
openi6x_1.9.0.zip for everyone else.

Is there a link to some documentation on this github with a header if one isn't willing to host one where one can find out how to use this firmware without having to search the internet themselves?

Not trying to put down the project just bring stuff up as a new user with limited knoledge.

**Comment by ajjjjjjjj** [comment]
https://doc.open-tx.org/manual-for-opentx-2-2/software-overview

heli - means you have access to model heli menu,
elrsv3 if you need ExpressLRS config tool.

Main project readme links to how to install.
